geopyspark/geotrellis/render.py

```diff
@@ -49,7 +54,7 @@ def get_hex(geopysc, ramp_name, num_colors=None):
         return list(geopysc._jvm.geopyspark.geotrellis.ColorRamp.getHex(ramp_name))
 
 
-class PngRDD(object):
+class PngRDD(RDDWrapper):
```

Comment:
Aside from being correctly located in a class hierarchy, I suppose not.  But looks like a duck, sounds like a duck...  If we weren't supposed to use the class structure, why bother having it?